wherehows-web/app/components/application/banner-alerts.ts

```diff
@@ -0,0 +1,49 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+import { computed, get, set } from '@ember/object';
+import ComputedProperty from '@ember/object/computed';
+import BannerService, { IBanner } from 'wherehows-web/services/banners';
+
+export default class BannerAlerts extends Component {
+  /**
+   * Imports the service used to handle actual activation and dismissal of banners. The service also
+   * maintains the banners list
+   * @type {Serivce}
+   */
+  banners: ComputedProperty<BannerService> = service();
```

Comment:
Can we handle this in the notifications service? I think that can be easily augmented to handle banners rather an creating a new service for what is essentially a notification? Thoughts?

Reply:
That was my initial thought as well, but when reading the code for the notification service it seemed already very specialized for modals and toasts, and also already has a lot going on. Two drawbacks I saw about combining banners into notifications were 1) there would be very little shared between them in terms of methods/properties and 2) taking 1 into account, it felt like it would be adding needly complexity to the notifications service and make it harder to differentiate what part of the functionality goes with what feature. 

How do you feel about that logic?